src/types/bee.ts

```diff
@@ -339,7 +340,20 @@ export interface IBeeConfig {
       handler: BeePluginContentDialogHandler<IRefreshSavedRow>
     }
   },
-  rowsConfiguration?: Record<string, unknown>
+  rowsConfiguration?: { 
+    defaultRows?: boolean,
+    emptyRows?: boolean,
+    externalContentURLs?: Array<{
+      name: string
+      value: string
+      id?: string | number
+      handle?: string
```

Comment:
`isLocal?: boolean` is another possible property for the `externalContentURLs` elements.
Also, `id?: string | number` is an unsupported property.
Consider the possibility to define a new interface `IExternalContentUrl` to be used here and also exported to be used elsewhere.

Reply:
> Consider the possibility to define a new interface `IExternalContentUrl` to be used here and also exported to be used elsewhere.

I think it will need a more specific name than that. There is another `externalContentURLs` config option (`contentDialog.externalContentURLs`) which accepts different properties.

- [📚 Content Dialog - Custom Rows](https://docs.beefree.io/content-dialog/#custom-rows)

Reply:
Hi @BenJenkinson thank you for your comment!

I agree that it can be confusing to have the same property name in the `contentDialog` object and also in the `rowsConfiguration` object.
I think it's appropriate to use `IExternalContentUrl` in this case and think about something different for the content dialog.